src/layouts/sidebar-sidecar/components/sidecar-scroll-container/sidecar-scroll-container.module.css

```diff
@@ -77,6 +77,7 @@
 
 .scrollContainer {
 	overflow-y: auto;
+	overflow-x: hidden;
```

Comment:
We need this to avoid the horizontal scrollbar if the dropdown disclosure active item text is really long. Since the sidecar items are truncated I don't believe this affects anything there but please let me know if there are concerns with this.

Reply:
💭 Tested this out and I think we may be able to get away with some other solution - looks like the scroll is being caused by the "disclosure activator" `<button />` `:before` pseudo-element, which I think is a shadow or something? I think we could set `overflow: hidden` on `variants-dropdown-disclosure_root` to scope the overflow fix to this use case 👍

https://github.com/hashicorp/dev-portal/assets/4624598/1596d851-b5ff-48f7-bfe3-b68d8b195e26

That being said, could also see the case for hiding all horizontal overflow in sidecar as done here... but worried it could hide a future quirk or bug we might otherwise notice 🤔 So probably either one is fine!

Reply:
Ooo I'll check out this suggestion! I'm definitely down for minimizing the radius of this change. TY

Reply:
Hmmm @zchsh if I set overflow hidden to the root, then the dropdown items aren't visible 🤔

Reply:
I'm going to roll with this for now but we should keep an eye out for any side effects. I've clicked around to see a bunch of sidecars and haven't noticed an issue yet.